src/server/scripts/EasternKingdoms/ZulGurub/boss_marli.cpp

```diff
@@ -77,142 +80,180 @@ enum Misc
 float const DamageIncrease = 35.0f;
 float const DamageDecrease = 100.f / (1.f + DamageIncrease / 100.f) - 100.f;
 
+float const MaxRangeHatch = 100.f;
```

Comment:
It's constant I came up with, if you know any reliable source I can take from I will be happy to change.